.github/workflows/ignore_words

```diff
@@ -0,0 +1,16 @@
+ans
+archtype
+copyable
+crate
+datas
+expext
```

Comment:
Why this is an exception? Do you know where use this word?

Reply:
It is used to ignore https://github.com/WasmEdge/WasmEdge/pull/2165#discussion_r1051371780
codespell's ignore words only support all lowercase words so `expext` actually means `ExpExt`

Reply:
Get it, can’t we write “ExpExt” here?